<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveItemFromInventory</class>
 <widget class="QWidget" name="RemoveItemFromInventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="TopLayout">
    <item>
     <layout class="QHBoxLayout" name="EntryLayout">
      <item>
       <widget class="QLabel" name="ItemCodeLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Item Code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ItemCodeEntry"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="ButtonLayout">
      <item>
       <widget class="QPushButton" name="RemoveItem">
        <property name="text">
         <string>Remove Item</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
